chemclipse/plugins/org.eclipse.chemclipse.chromatogram.xxd.filter.supplier.savitzkygolay/src/org/eclipse/chemclipse/chromatogram/xxd/filter/supplier/savitzkygolay/settings/ChromatogramFilterSettings.java

```diff
@@ -28,6 +29,9 @@
 	@JsonProperty(value = "Width", defaultValue = "5")
 	@IntSettingsProperty(minValue = PreferenceSupplier.MIN_WIDTH, maxValue = PreferenceSupplier.MAX_WIDTH)
 	private int width = 5;
+	@JsonProperty(value = "Apply Filter individual to all ion channels", defaultValue = "false")
+	@JsonPropertyDescription(value = "Per Ion Filter Calculation.")
```

Comment:
"Filter individual channels" or something like that seems more suitable as a name